source/chapters/preferences.rst

```diff
@@ -240,6 +240,47 @@ You can manually add, relink, and remove Mixxx music directories in
           rescan afterwards. Select :menuselection:`Library --> Rescan Library`
           in the menu.
 
+.. _configuration-colors:
+
+Colors
+======
+
+.. _configuration-colors-palette:
+
+Selecting Palettes and Default Colors
+-------------------------------------
+
+**Track/Hotcue Palette**
+  It is possible to assign colors to tracks and :ref:`hotcues <interface-hotcues>`
+  to highlight them. Mixxx offers a selection of different palettes that you can
+  choose colors from.
+
+  You can also make your own palette by clicking the :guilabel:`Edit` button next
+  to the palette name. This will open a new window which allows you to add, modify,
+  reorder and remove colors in the palette and save it under a new name.
+
+  When editing the hotcue palette, the palette editor also allows assigning a
+  specific color to a hotcue number. This setting honored when
+  :guilabel:`Hotcue default color` is set to `By hotcue number`. If no color in
+  the palette has a hotcue number assigned to it, the color of new hotcues will
+  be mapped to the position of the color in the palette.
```

Comment:
> [...] the color of new hotcues will be mapped to the position of the color in the palette.

I personally don't find this sentence very clear.

Reply:
This is explained in the next section. Maybe this sentence here can be removed.

Reply:
I improved the explanation and added an example.